src/pages/publication/DescriptionPanel.tsx

```diff
@@ -170,28 +169,25 @@ const DescriptionPanel: FC<DescriptionPanelProps> = ({ goToNextTab, savePublicat
           <StyledFieldHeader>{t('description.project_association')}</StyledFieldHeader>
 
           <StyledFieldWrapper>
-            <FieldArray name={DescriptionFieldNames.PROJECTS}>
-              {({ name, insert, remove }: FieldArrayRenderProps) => (
+            <Field name={DescriptionFieldNames.PROJECT}>
```

Comment:
skal det ikke kunne være flere prosjekter lenger? ser det fortsatt er listet opp flere i wireframes

Reply:
ikke støtte for det i backend hvertfall. mener det ble sagt at i de aller fleste tilfeller, så er det kun et prosjekt tilknyttet.

Reply:
står ingenting om at det skal være flere prosjekt her: https://github.com/BIBSYSDEV/NVA-example-functional-requirements/blob/master/mvp.feature

kun features for å legge til ett prosjekt